examples/user_guide/05-Building_Composite_Objects.ipynb

```diff
@@ -389,7 +389,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The second form demonstrates HoloViews' **deep indexing** feature, which allows indexes to cross nested container boundaries. The above is as far as we can index before reaching a heterogeneous type (the ``Overlay``), where we need to use attribute access. Here is the more explicit method of indexing down to a curve, using ``.select`` to specify dimensions by name instead of bracket-based indexing by position:"
+    "The second form demonstrates HoloViews' **deep indexing** feature, which allows indices to cross nested container boundaries. The above is as far as we can index before reaching a heterogeneous type (the ``Overlay``), where we need to use attribute access. Here is the more explicit method of indexing down to a curve, using ``.select`` to specify dimensions by name instead of bracket-based indexing by position:"
```

Comment:
I'm not sure indices would be preferred here: http://grammarist.com/usage/indexes-indices/

Reply:
Personally, ``indices`` seems right to me but I don't mind if you push a change to revert.

Reply:
That's because you are British (see the link). :-)  I'm just saying, both versions are perfectly good, so there's no reason to flip-flop between them...

Reply:
The PR has been merged but I'll also merge a PR if you want to submit a change flipping it back :-)